RELEASE.md

```diff
@@ -23,7 +23,9 @@ In order to create a new release of the `cloudnative-pg` chart, follow these ste
 
 1. Take note of the current value of the release: see `.version` in `charts/cloudnative-pg/Chart.yaml`
     ```bash
-    yq -r '.version' charts/cloudnative-pg/Chart.yaml
+    OLD_VERSION=$(yq -r '.version' charts/cloudnative-pg/Chart.yaml)
+    OLD_CNPG_VERSION=$(yq -r '.appVersion' charts/cloudnative-pg/Chart.yaml)
+    echo $OLD_VERSION
```

Comment:
I don't think a release PR is the place to update the current release instructions =) probably this should go into the release.sh script and be automated as is suggested down

Reply:
It's still a manual process. You might be updating the chart with or without an operator release. I made this so that you can just go through the list, run them if you need to and verify the result is what you intended.

I updated the cluster chart one a while ago, but never the operator one.

Reply:
I agree with @sxd, I'd keep the changes to the release procedure in a separate PR. The rest looks good to me.